core/src/trezor/wire/errors.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, message: str) -> None:
 
 
 class DataError(Error):
-    def __init__(self, message: str) -> None:
+    def __init__(self, message: str = "Data error") -> None:
```

Comment:
see above, there should be no "generic data errors" without a reason

Reply:
https://github.com/trezor/trezor-firmware/pull/3534/commits/339da2725da19a08e116a30fa035fcb17f31a463